### PR TITLE
tesseract: add page

### PR DESCRIPTION
**Pull request by adius**

### CODE REVIEW

File: pages/common/tesseract.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Tesseract
+
+> OCR (Optical Character Recognition) engine.
+
+- Recognize text in image `image.png` and save to `content.txt` file:
+
+`tesseract {{image.png}} {{content}}`
+
+- Specify another language than English:
```
Comment (agnivade) [inline pages/common/tesseract.md]: Please mention that deu means German.

File: pages/common/tesseract.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Tesseract
+
+> OCR (Optical Character Recognition) engine.
+
+- Recognize text in image `image.png` and save to `content.txt` file:
```
Comment (agnivade) [inline pages/common/tesseract.md]: Its better to rename the file as output.txt, and put the same name in the example too. Makes it uniform with other pages that we have.

**Comment by adius** [comment]
@agnivade Fixed the issues!

**Comment by waldyrious** [comment]
@adius I made some changes to the page -- please take a look and let me know what you think.

**Comment by adius** [comment]
Well except for that it MUST be `output` and not `output.txt` good.

To use your changes and keep it right:

```
- Recognize text in an image and save it to the text file `output.txt`:

`tesseract {{image.png}} {{output}}`
```

**Comment by adius** [comment]
@waldyrious Still there?

**Comment by agnivade** [comment]
@adius - He seems to be absconding for a few days. :stuck_out_tongue_closed_eyes:

Probably busy with something.

**Comment by agnivade** [comment]
@adius - Let me know if this seems alright.

**Comment by adius** [comment]
@agnivade Looks good  to me. ...and good idea to explicitly mention that a file extension must not be specified!

**Comment by agnivade** [comment]
We always try to be as explicit and clear as possible whenever there are chances of confusion. It sure is not intuitive that to generate a file `output.txt`, one has to enter `output` in the arguments. So, its better to mention it outright.

**Comment by waldyrious** [comment]
Thanks guys. Sorry I went MIA.

**Pull request merged**